src/Models/UserModel.php

```diff
@@ -118,7 +118,7 @@ public function addToDefaultGroup(User $user): void
     public function fake(Generator &$faker): User
     {
         return new User([
-            'username' => $faker->userName,
+            'username' => str_replace('.', ' ', $faker->userName),
```

Comment:
Was this necessary? IMO Shield should support periods in usernames and *not* spaces.

Reply:
If so, we need to change the validation rule for registration.
https://github.com/codeigniter4/shield/blob/b0ccc14d5dda68df74130723227fbc0370d88221/src/Controllers/RegisterController.php#L134

Reply:
I'm speaking anecdotally, I don't have an RFC reference or anything, but I would never anticipate allowing a space in a username and I also know many services that allow periods, especially so you can use Gmail-style credentials: `matthew.gatner@gmail.com => matthew.gatner`

Reply:
@MGatner Why don't we merge this, and after that, how about discussing the username validation rules?
This PR is a bit old, and Registration is out of scope of this PR.